modules/hooks/settings.js

```diff
@@ -18,14 +18,14 @@ export default function() {
                 })
             }
             // Update counter values, can't just use ui.combat because there might be popped out combat trackers
-            [ui.combat].concat(Object.values(ui.windows).filter(w => w instanceof CombatTracker)).forEach(tracker => {
+            let trackers = [ui.combat].concat(Object.values(ui.windows).filter(w => w instanceof CombatTracker));
```

Comment:
that's actually wierd - but i noticed that sometimes, group advantage updates breaks using original code - it was throwing exception at [ui.combat].concat()...().foreach(). no idea why.